client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail/workflow-detail.component.ts

```diff
@@ -121,8 +120,9 @@ export class WorkflowDetailComponent extends BaseMeetingComponent {
 
     private set workflow(workflow: ViewMotionWorkflow) {
         this._workflow = workflow;
+        console.log(`Workflow set:`, workflow);
```

Comment:
Could it be that you forgot to remove this?

Reply:
I'll remove this line.